src/templates/Article.jsx

```diff
@@ -5,11 +5,54 @@ import Helmet from 'react-helmet';
 
 import Breadcrumbs from '../templateComponents/Breadcrumbs.jsx';
 
+import titleify from '../../utils/titleify';
+
 const propTypes = {
   data: PropTypes.object,
   location: PropTypes.object
 };
 
+function getOgTitle(pathname) {
+  const pathMap = pathname
+    // remove leading and trailing slash
+    .replace(/^\/([a-z0-9/-]+[^/])\/?$/i, '$1')
+    .split('/');
+
+  let metaTitle = '';
+  pathMap.forEach((title, i) => {
+    if (i === pathMap.length - 1) {
+      metaTitle += titleify(title);
+    } else {
+      metaTitle += titleify(title + ' | ');
+    }
+  });
+
+  return metaTitle;
+}
+
+function getOgDescription(html) {
+  const el = document.createElement('html');
```

Comment:
We should be able to get an extract from graphql. Copy the page query at the end of the file and have a play around with the graphql server is which is available in development.